### PR TITLE
 Book store Basic setup

### PR DESCRIPTION
**Pull request by codeWithHyder**
Basic book store setup: add routes to website and basic setup

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Book = () => (
+  <>
+    <h1 className="newbook">New book is being added soon stay tuned!</h1>
+    {/* <h4>Book Author</h4> */}
```
Comment (rustam-tolipov) [inline src/components/Book.js]: - [x] To keep your code clean. Please remove all unused commented codes from your project.

File: src/components/NewBookEntry.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        type="text"
+        placeholder="Add a new book..."
+        id="newbook"
+        name="bookname"
+      />
+      <select className="author">
+        <option readOnly>Author</option>
+        <option>abc</option>
+        <option>xyz</option>
+      </select>
```
Comment (rustam-tolipov) [inline src/components/NewBookEntry.js]: - [x] In the project's requirement, we must use the input to get the author. Kindly get the author with input. You can keep your selection for the genres of your book.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @codeWithHyder,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great readme description_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <>
+    <p>Code is being implemented to this part soon!</p>
+  </>
```
Comment (jerryowusu) [inline src/components/Categories.js:6]: [OPTIONAL] As part of the project requirements, this component should contain a `<Button>` with the text "Check status". :wink:

**Review by jerryowusu — APPROVED** [review]
Hi @codeWithHyder, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Used react-router-dom :+1: 
Initialized project with Create react app :+1: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**